earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -129,6 +145,12 @@ main(int argc, char *argv[]) {
       usage(argv[0], "--historical_imagery is not a valid option for mercator "
           "imagery projects.");
     }
+    if ((enable_terrain_overlay || disable_terrain_overlay || start_level != 0 ||
```

Comment:
if imagery project we may need to look for those arguments as they were not valid to be used and hence we would not need this code ?

Reply:
I'm not sure I understand your concern here. This is the code that is checking to make sure those options aren't being passed in when using one of the imagery commands.

Reply:
we don't need to validate those arguments if they were not to be used. We can just ignore them. It's the same situation if some one pass some arguments that are not part of the command, we can't know what those are, unless if we want to exclude any extra argument and in that case we can do it for any thing not expected and not just the overlay terrain.

Reply:
Okay, I see what you're saying. There's already code later in the function that prevents the terrain options from being used for a non-terrain project, so I guess there's no real need to have this here.
Especially since the `--help` text has been updated to only show applicable options.